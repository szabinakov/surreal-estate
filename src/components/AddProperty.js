import React from 'react'

const AddProperty = () => {
    return (
        <div>Add Property</div>
    )
}

export default AddProperty