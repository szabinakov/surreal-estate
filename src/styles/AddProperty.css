.AddProperty {
    display: flex;
    justify-content: center;
    margin-top: 30px;
}

.FormContainer{
    width: 300px;
    border: 1px solid black;
    text-align: center;
}
.FormInput,.FormSelectCity,.FormSelectType,.FormRoom,.FormMaximumPrice, .FormEmail,.FormBathroom  {
    display: flex;
    flex-direction: column;
    padding: 5px;
    margin: 10px;
}
