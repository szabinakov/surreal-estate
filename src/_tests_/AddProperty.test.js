import React from 'react';
import { render } from '@testing-library/react';
import AddProperty from '../components/AddProperty';

describe('AddProperty', () => {
    it('renders correctly', () => {
        const {asFragment} = render (<AddProperty/>)

        expect(asFragment).toMatchSnapshot()
    })
})
