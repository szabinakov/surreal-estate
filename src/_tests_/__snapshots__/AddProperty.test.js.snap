// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddProperty renders correctly 1`] = `[Function]`;
